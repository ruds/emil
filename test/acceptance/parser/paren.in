();
({foo=1, bar=baz});
(foo, op ++);
(foo; op ++);
(foo,);
(foo;);
(;);
(,);
(foo, bar; baz);
(foo; bar, baz);
