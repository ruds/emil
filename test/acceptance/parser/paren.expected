0001 (ExprTopDecl
         :expr
         (UnitExpr))
0002 (ExprTopDecl
         :expr
         (RecordExpr
             :rows
             (
                 (RecRowSubexpr
                     :label
                     "666F6F"
                     :val
                     (BigintLiteralExpr
                         :val
                         1))
                 (RecRowSubexpr
                     :label
                     "626172"
                     :val
                     (IdentifierExpr
                         :qualifiers
                         ()
                         :identifier
                         "62617A"
                         :is_op
                         false
                         :is_prefix_op
                         false))
             )))
0003 (ExprTopDecl
         :expr
         (TupleExpr
             :exprs
             (
                 (IdentifierExpr
                     :qualifiers
                     ()
                     :identifier
                     "666F6F"
                     :is_op
                     false
                     :is_prefix_op
                     false)
                 (IdentifierExpr
                     :qualifiers
                     ()
                     :identifier
                     "2B2B"
                     :is_op
                     true
                     :is_prefix_op
                     false)
             )))
0004 (ExprTopDecl
         :expr
         (SequencedExpr
             :exprs
             (
                 (IdentifierExpr
                     :qualifiers
                     ()
                     :identifier
                     "666F6F"
                     :is_op
                     false
                     :is_prefix_op
                     false)
                 (IdentifierExpr
                     :qualifiers
                     ()
                     :identifier
                     "2B2B"
                     :is_op
                     true
                     :is_prefix_op
                     false)
             )))
0005 ERROR
0006 ERROR
0007 ERROR
0008 ERROR
0009 ERROR
0009 ERROR
0010 ERROR
0011 (EndOfFileTopDecl)
